.github/workflows/consecutive.yml

```diff
@@ -63,154 +98,60 @@ jobs:
-  #       with:
-  #         folder-name: access
-  #         jobsNumber: $JOBS_NUMBER
-
-  #     - uses: ./.github/actions/run-process-upload
-  #       if: always()
-  #       timeout-minutes: 60
-  #       with:
-  #         folder-name: finance
-  #         jobsNumber: $JOBS_NUMBER
-
-  #     - uses: ./.github/actions/run-process-upload
-  #       if: always()
-  #       timeout-minutes: 60
-  #       with:
-  #         folder-name: governance
-  #         jobsNumber: $JOBS_NUMBER
-
-  #     - uses: ./.github/actions/run-process-upload
-  #       if: always()
-  #       timeout-minutes: 60
-  #       with:
-  #         folder-name: helpers
-  #         jobsNumber: $JOBS_NUMBER
-
-  #     - uses: ./.github/actions/run-process-upload
-  #       if: always()
-  #       timeout-minutes: 60
-  #       with:
-  #         folder-name: metatx
-  #         jobsNumber: $JOBS_NUMBER
-
-  #     - uses: ./.github/actions/run-process-upload
-  #       if: always()
-  #       timeout-minutes: 60
-  #       with:
-  #         folder-name: proxy
-  #         jobsNumber: $JOBS_NUMBER
-
-  #     - uses: ./.github/actions/run-process-upload
-  #       if: always()
-  #       timeout-minutes: 60
-  #       with:
-  #         folder-name: security
-  #         jobsNumber: $JOBS_NUMBER
-
-  node-contracts-02:
-    runs-on: neon-hosted
-    timeout-minutes: 361
-    if: always()
-    # experimental changes
-    # needs: node-contracts-01
+  openzeppelin:
+    timeout-minutes: 360
+    runs-on: self-hosted
+    needs:
+      - prepare-env
+    env:
+      JOBS_NUMBER: ${{ needs.prepare-env.outputs.jobs }}
+      NETWORK: ${{ needs.prepare-env.outputs.network }}
     steps:
       - uses: actions/checkout@v2
-      # TODO: de-duplicate it https://github.com/neonlabsorg/neon-compatibility/issues/155
-      - name: Display input parameters
-        run: |
-          if [ -n "${{ github.event.inputs.jobsNumber }}" ]; then
-            echo "JOBS_NUMBER=${{ github.event.inputs.jobsNumber }}" >> $GITHUB_ENV
-          fi
-          echo "cat $GITHUB_ENV"
-          cat $GITHUB_ENV
-        shell: bash
-      # macos-11
-      # - uses: ./.github/actions/allure-prep
       - uses: ./.github/actions/openzeppelin-preparation
-      - uses: ./.github/actions/switch-network
-        if: always()
+      - name: Switch network
+        run: |
+          cp "./.env.$NETWORK" "./openzeppelin-contracts/.env"
       - uses: ./.github/actions/allure-environment-data
-        if: always()
       - name: Use Node.js 16
         uses: actions/setup-node@v2
-        if: always()
         with:
           node-version: 16.x
           cache: "npm"
       - run: npm ci
         if: always()
       - run: npm run build --if-present
+      - name: "Run openzeppelin in parallel"
+        run: |
+          cd openzeppelin-contracts
+          cp ../run_parallel.py ./
+          cp ../rrrun.sh ./
+          chmod +x rrrun.sh
+          npx hardhat compile
+          python3 run_parallel.py
```

Comment:
why do we need this instead of GNU parallel?

Reply:
command in python script - parallel. This script uses for run only N copies tests and split all test files for N processes. Not every file in new node.js